docs/_asset/root.client.js

```diff
@@ -0,0 +1,15 @@
+import {Suspense} from 'react'
+
+export function Root(props) {
+  const {response} = props
```

Comment:
```suggestion
export function Root({response}) {
```

Reply:
I find that your suggestion makes it harder to debug code

Reply:
I respectfully disagree that it is harder to debug.
And would also note that this pattern is used elsewhere in this PR.

Reply:
Two examples where I find it hard to debug:
- error messages, as there's no name of the parameter
- Logging what else gets passed in props, because there is no props

Do you experience these things and deal with them differently?

And whether being inconsistent: yeah probably 😬😅

Reply:
I usually handle that by dropping a breakpoint in the debugger to see what is going on at that point in time.
And for React components specifically, the react dev tools will show what props are passed in.

Reply:
hmm, I never use either of those